src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php

```diff
@@ -0,0 +1,341 @@
+    {
+        $script = $this->clickByTitle('.column-navigation .item-text', $itemTitle);
+        $this->getSession()->executeScript($script);
+    }
+
+    /**
+     * @Then I double click the column navigation item :itemTitle
+     */
+    public function iDoubleClickTheColumnNavigationItem($itemTitle)
+    {
+        $script = $this->clickByTitle('.column-navigation .item-text', $itemTitle, 'dblclick');
+        $this->getSession()->executeScript($script);
+    }
+
+    /**
+     * Expect until all of the named events have been fired
+     * @Then I expect the following events:
+     */
+    public function iExpectTheFollowingEvents(PyStringNode $eventNames)
+    {
+        $this->waitForAuraEvents($eventNames->getStrings(), 5000);
+    }
+
+    /**
+     * @Then I expect the ":eventName" event
+     */
+    public function iExpectTheEvent($eventName)
+    {
+        $this->waitForAuraEvents(array($eventName));
+    }
+
+    /**
+     * @Then I wait a second for the ":eventName" event
+     */
+    public function iWaitASecondForTheEvent($eventName)
+    {
+        $this->waitForAuraEvents(array($eventName), 1000);
+    }
+
+    /**
+     * @Then there should be :targetNb form errors
+     */
+    public function thereShouldBeErrors($targetNb)
+    {
+        $nbErrors = $this->getSession()->evaluateScript("$('.husky-validate-error').length");
+        if ($nbErrors != $targetNb) {
+            throw new \Exception(sprintf(
+                'Was expecting "%s" form errors, but got "%s"', $targetNb, $nbErrors
+            ));
+        }
+    }
+
+    /**
+     * @Given I expect the aura component ":name" to appear
+     */
+    public function iExpectTheAuraComponentToAppear($name)
+    {
+        $selector1 = 'div[data-instance-name=\\"' . $name . '\\"]';
+        $selector2 = 'div[data-aura-instance-name=\\"' . $name . '\\"]';
+        $this->getSession()->wait(self::LONG_WAIT_TIME, sprintf('$(\'%s\').children().length > 0 || $(\'%s\').children().length > 0', $selector1, $selector2));
+        $this->assertAtLeastOneSelectors(array($selector1, $selector2));
+    }
+
+    /**
+     * Fill in the named husky field. Husky fields may not use standard HTML
+     * inputs, so they need some special handling.
+     *
+     * @param string $name Name of field to fill in 
+     * @param string $value Value to fill in
+     * @param string $parentSelector Optional parent selector
+     */
+    private function fillInHuskyField($name, $value, $parentSelector = '')
+    {
+        foreach (array(
+            'data-aura-instance-name',
+            'data-mapper-property'
+        ) as $propertyName) {
+            $script = <<<EOT
+    $('%s[%s="%s"]').data('element').setValue('%s');
+EOT
+            ;
+
+            $script = sprintf($script, $parentSelector, $propertyName, $name, $value);
+            try {
+                $this->getSession()->executeScript($script);
+                return;
+            } catch (\Exception $e) {
```

Comment:
We could catch the WebDriverException bit it isn't very specific. It doesn't tell you that this is a JS exception for intance.